core/async/src/examples/simple.rs

```diff
@@ -0,0 +1,64 @@
+use std::{sync::Arc, time::Duration};
+
+use derive_enum_from_into::{EnumFrom, EnumTryInto};
+
+use crate::{
+    messaging::Sender,
+    test_loop::{CaptureEvents, LoopEventHandler, TestLoopBuilder, TryIntoOrSelf},
+};
+
+use super::sum_numbers::{ReportSumMsg, SumNumbersComponent, SumRequest};
+
```

Comment:
I think I need a higher level description about the structure of the examples dir ;-) 
is the idea that we have a 'sum_numbers' component and then we have simple.rs just to test it?

(if yes - I'd rename the simple.rs to sum_numbers_test.rs)

Reply:
Good idea, let me rename it like that. I don't really have a structure of the examples dir yet, I'm making it up as we go :)